### DISCUSSION TITLE
accessing the database changes

### DISCUSSION THREAD

**Discussion by dbarrosop** [nhost/nhost#2710] [General]
Hello all,

The TLDR is that public access to the database will be disabled by default and will have to be enabled explicitly for those that need it. If you are not using the connection string and haven't used it in the last 30 days this doesn't affect you, otherwise, keep reading.

We wanted to let you know we are introducing a new mechanism to control access to your database publicly. You can find the documentation [here](https://docs.nhost.io/guides/database/access). 
To minimize impact we have looked into who has used the connection string in the last 30 days and enabled public access directly, so things should still be working for you. However, we'd encourage you to do two things:

1. Verify under settings -> database that public access is enabled
2. If you are using the toml; update the CLI and run `nhost config pull` or follow the instructions [here](https://docs.nhost.io/guides/database/access)

Otherwise, you might lose public access to your database on your next deploy.

We apologize for any inconvenience this change may lead to.

If you have questions/concerns don't hesitate to let us know.

Regards
